### PR TITLE
NLU response added

### PR DESCRIPTION
**Pull request by RiyaMRoy04**

### CODE REVIEW

**Review by sanjeevghimire — CHANGES_REQUESTED** [review]
* Readme is broken. I think its due to the merge
*  the code needs to be tested deploying to kube cluster in ibm cloud and locally.

**Comment by sanjeevghimire** [comment]
the NLU icon on arch diagram is blurry as well

**Comment by RiyaMRoy04** [comment]
@sanjeevghimire updated README with the clear arch diagram

**Comment by stevemar** [comment]
I think this is fine, might be worth one more walkthrough @sanjeevghimire and you can tweak anything else you might notice.

**Review by stevemar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,25 +1,29 @@
+[![Build Status](https://travis-ci.org/IBM/<repo>.svg?branch=master)](https://travis-ci.org/IBM/<repo>)
```
Comment (stevemar) [inline README.md:1]: this needs to be removed. it doesn't link to anything.

Comment (RiyaMRoy04) [reply README.md:1]: @stevemart Lint and Travis is WIP 👍

Comment (stevemar) [reply README.md:1]: I figured :)

But we can add that line when linting and travis are actually up :)

**Comment by sanjeevghimire** [comment]
going through it one more time and testing deploy to kube cluster. Will move to publishing after this.

**Pull request merged**